core/src/main/java/com/datastax/oss/driver/api/core/tracker/RequestTracker.java

```diff
@@ -75,6 +94,23 @@ default void onError(
       @NonNull DriverExecutionProfile executionProfile,
       @Nullable Node node) {}
 
+  /**
+   * @deprecated This method only exists for backward compatibility. Override {@link
+   *     #onError(Request, Throwable, long, DriverExecutionProfile, Node, String, ExecutionInfo)}
+   *     instead.
+   */
+  @Deprecated
+  default void onError(
```

Comment:
Similar to the newly deprecated `onSuccess`, this one is still being used by `GraphRequestHandler` and `ContinuousRequestHandlerBase`